lib/pages/settings_dashboard/settings/settings.dart

```
@@ -86,26 +88,56 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
         OkCancelResult.cancel) {
       return;
     }
-    final matrix = Matrix.of(context);
     if (PlatformInfos.isMobile) {
-      await tryLogoutSso(context);
+      await _logoutActionsOnMobile();
+    } else {
+      await _logoutActions();
     }
-    if (matrix.twakeSupported == true) {
-      final hiveCollectionToMDatabase = getIt.get<HiveCollectionToMDatabase>();
-      await hiveCollectionToMDatabase.clear();
+  }
+
+  Future<void> _logoutActionsOnMobile() async {
+    try {
+      await tryLogoutSso(context);
+    } catch (e) {
+      Logs().e('SettingsController()::_logoutActionsOnMobile - error: $e');
+      return;
```

IMO, should we throw it, instead of swallow it